client/src/containers/Economy/components/GDP/index.tsx

```diff
@@ -19,6 +19,26 @@ export const GDP = ({ onData }) => {
     });
   };
 
+  const changeNumberFormat = (val) => {
+    
+    // Nine Zeroes for Billions
+    const changedNum =  Math.abs(Number(val)) >= 1.0e+9
+
+    ? (Math.abs(Number(val)) / 1.0e+9).toFixed(2) + " billion"
```

Comment:
Don't use ternary operator inside ternary operator.
Follow https://eslint.org/docs/latest/rules/no-nested-ternary#rule-details
Consider adding this rule to ESLint